### PR TITLE
Hide inventory buttons and fix css of disabled button [SCI-4810]

### PR DESCRIPTION
**Pull request by mlorb**
Jira ticket: [SCI-4810](https://biosistemika.atlassian.net/browse/SCI-4810)

### What was done
-Hide inventory buttons
-fix css of disabled button

### CODE REVIEW

File: app/assets/javascripts/repositories/index.js [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
+        $('.main-actions [data-action-mode="single"]').removeClass('disabled hidden');
+        $('.main-actions [data-action-mode="multiple"]').removeClass('disabled hidden');
         $('#renameRepoBtn').attr('href', row.data('rename-modal-url'));
         $('#deleteRepoBtn').attr('href', row.data('delete-modal-url'));
         $('#copyRepoBtn').attr('href', row.data('copy-modal-url'));
         break;
       default:
-        $('.main-actions [data-action-mode="single"]').addClass('disabled');
-        $('.main-actions [data-action-mode="multiple"]').removeClass('disabled');
+        $('.main-actions [data-action-mode="single"]').removeClass('hidden').addClass('disabled');
```
Comment (radiokills) [inline app/assets/javascripts/repositories/index.js:28]: do we really need 'addClass('disabled');'

File: app/assets/stylesheets/repository/index.scss [hunk 2]
```diff
@@ -11,6 +11,12 @@
     }
   }
 
+  .toolbar {
+    .hidden {
+      display: none;
+    }
+  }
+
```
Comment (radiokills) [inline app/assets/stylesheets/repository/index.scss:19]: Isn't there a similar style already defined? I would imagine it should already be somewhere, maybe even as a general offering from bootstrap.

File: app/assets/javascripts/repositories/index.js (hunk 1 above)
Comment (aignatov-bio) [reply app/assets/javascripts/repositories/index.js:28]: We have actions against only one item.

File: app/assets/stylesheets/repository/index.scss (hunk 2 above)
Comment (aignatov-bio) [reply app/assets/stylesheets/repository/index.scss:19]: Yes it is bootstrap global class

File: app/assets/stylesheets/shared_styles/elements/buttons.scss [hunk 3]
```diff
@@ -137,13 +137,17 @@
     }
   }
 
+  &.disabled,
   &:disabled {
-    border: $border-tertiary;
     color: $color-silver-chalice;
+    cursor: not-allowed;
     opacity: .5;
+    pointer-events: auto !important;
```
Comment (aignatov-bio) [inline app/assets/stylesheets/shared_styles/elements/buttons.scss:145]: Why it here?

Comment (mlorb) [reply app/assets/stylesheets/shared_styles/elements/buttons.scss:145]: to overwrite bootstrap

Comment (aignatov-bio) [reply app/assets/stylesheets/shared_styles/elements/buttons.scss:145]: You will break boostrap

Comment (aignatov-bio) [reply app/assets/stylesheets/shared_styles/elements/buttons.scss:145]: It is important for 'disabled' links, don't do it

Comment (mlorb) [reply app/assets/stylesheets/shared_styles/elements/buttons.scss]: :+1:

File: app/assets/javascripts/repositories/index.js (hunk 1 above)
Comment (radiokills) [reply app/assets/javascripts/repositories/index.js:28]: :+1:

**Pull request merged**